packages/core/src/models/uischema.ts

```diff
@@ -249,3 +258,15 @@ export const isGroup = (layout: Layout): layout is GroupLayout =>
 
 export const isLayout = (uischema: UISchemaElement): uischema is Layout =>
   (uischema as Layout).elements !== undefined;
+
+export const isScopeable = (obj: object): obj is Scopable =>
+  obj !== undefined && obj.hasOwnProperty('scope');
```

Comment:
This does not fit. In `Scopable` the `scope` is optional. So why check for it here?

Reply:
You are right it. The check is only useful if was

interface Scopable {
scope: string|null;
}